index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer"
+    />
+    <title>Medical Core</title>
+
+</head>
+
+<body class="body">
+    <header class="Header">
+
+        <nav class="nav">
+            <div class="">
+                <img src="img/Medical Core.png" alt="" class="nav__logo">
+
+            </div>
+            <ul class="nav-list">
+                <li class="nav__item">
+                    <a href="/#" class="nav__link"> Home</a></li>
+                <li class="nav__item"><a href="/#" class="nav__link"> Product</a></li>
+                <li class="nav__item"><a href="/#" class="nav__link"> Pricing</a></li>
+                <li class="nav__item"><a href="/#" class="nav__link"> Contact</a></li>
+            </ul>
+        </nav>
+        <div class="buttons buttons--spacebtn">
+            <a href="#" class="button ">Login</a>
+            <a href="#" class="button button--bg-color">Join Us <span class=" arrow">  &#8594;</span></a>
```

Comment:
## naming wrong
`button--bg-color` Naming the modifiers corresponding to their CSS representation is not recommended.

`arrow` why you can make a class that you will not re-use. makes it element or modifier